### PR TITLE
Add example for radio in proper place

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #702
[Problem] Wrong radio example
[Solution] Writing new example

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+		Wearable UI
+	</title>
+	<link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
```
Comment (TomaszLukawskiSam) [inline examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: you have to change paths to tau:
```
    <link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
    <link href="../../../lib/tau/wearable/theme/default/tau.circle.css" rel="stylesheet" />
    <link href="../../../css/style.css" rel="stylesheet" />
    <link href="../../../css/style.circle.css" rel="stylesheet" />
    <script src="../../../js/circle-helper.js">
    </script>
    <script src="../../../lib/tau/wearable/js/tau.js">
```

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <header class="ui-header">
+            <h2 class="ui-title">Radio</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+                <li class="li-has-radio">
+                    <label>Radio on <input type="radio" checked="checked" /></label>
+                </li>
+                <li class="li-has-radio">
+                    <label>Radio off<input type="radio"></label>
```
Comment (TomaszLukawskiSam) [inline examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: I know that this is example but when I tested this PR all radios are checked.

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+</head>
+<body>
+    <div class="ui-page">
+        <header class="ui-header">
+            <h2 class="ui-title">Radio</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+                <li class="li-has-radio">
+                    <label>Radio on <input type="radio" checked="checked" /></label>
```
Comment (TomaszLukawskiSam) [inline examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: lack of `name` attribute common for radio buttons.

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html (hunk 1 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Done

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html (hunk 3 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Done

**Review by lslachciak — COMMENTED** [review]
Some comments
this example works wrong on emulator
1. automatically content is scrolled to radio on (disabled)
image
2. unable to press radio off

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+</head>
+
+<body>
+    <div class="ui-page">
+        <header class="ui-header">
+            <h2 class="ui-title">Radio</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+                <radiogroup>
```
Comment (singa2000) [inline examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: I have a question.
What's the "radiogroup"?
Is it needed for sample?

Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Thank you for your advice. I removed this tag. It wasn't necessary.

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html (hunk 2 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Done!

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+</head>
+
+<body>
+    <div class="ui-page">
+        <header class="ui-header">
+            <h2 class="ui-title">Radio</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+                    <li class="li-has-radio">
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: check to indent.

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                        <label>
+                            Radio on
+                            <input type="radio" checked="checked" name="radio-el" id="firstRadio" />
+                        </label>
+                    </li>
+                    <li class="li-has-radio">
+                        <label>Radio off<input type="radio" name="radio-el" /></label>
```
Comment (DohyungLim) [inline examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Move the input tag to the next line.

**Review by DohyungLim — COMMENTED** [review]
2 comment.

**Comment by lslachciak** [comment]
Latest commit works properly on the emulator but please check @DohyungLim comments

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html (hunk 5 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Done

File: examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html (hunk 6 above)
Comment (korneliakobiela) [reply examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html]: Done.

**Comment by korneliakobiela** [comment]
I've updated example and resolve merge conflicts

File: examples/wearable/UIComponents/contents/user-inputs-elements/index.html [hunk 7]
```diff
@@ -38,6 +38,10 @@ <h2 class="ui-title">User Inputs Elements</h2>
                 <li>
                     <a href="pickers/index.html">Pickers</a>
                 </li>
+
```
Comment (singa2000) [inline examples/wearable/UIComponents/contents/user-inputs-elements/index.html]: Now radio.html file is located in the  contents\user-inputs-elements\controls\radio.html.
The radio, checkbox, onoff-switch is located in the contents\user-inputs-elements\controls folder. 
How about the move the file into "controls" folder  or remove the contents\user-inputs-elements\controls\radio.html, please.

**Review by singa2000 — APPROVED** [review]
It looks good.

**Comment by lslachciak** [comment]
@singa2000 @korneliakobiela 
Sorry but example app menu structure is now
User Input Elements --> Controls --> Radio/Checkbox/On/Off switch
Guide didn't assume additional controls category. And we agreed that sample app follow structure of guide.
I think that all those three items should be directly in User Input Elements
If you wish we can merge this PR but another PR will be needed to fix this

**Review by lslachciak — APPROVED** [review]
After more checks I think we need to think more about menu structure. Let's merge this PR as it is now

**Review by TomaszLukawskiSam — APPROVED** [review]
looks good to me.

**Pull request merged**